protocols/virtualfund/virtualfund_single_hop_test.go

```diff
@@ -490,3 +525,27 @@ func compareSignedProposals(a, b consensus_channel.SignedProposal) bool {
 		),
 	)
 }
+
+func compareObjectives(a, b Objective) string {
+	return cmp.Diff(&a, &b,
+		cmp.AllowUnexported(
+			Objective{},
+			channel.Channel{},
+			big.Int{},
+			state.SignedState{},
+			consensus_channel.ConsensusChannel{},
+			consensus_channel.Vars{},
+			consensus_channel.LedgerOutcome{},
+			consensus_channel.Balance{},
+		),
+	)
+}
+
+func compareGuarantees(a, b consensus_channel.Guarantee) string {
+	return cmp.Diff(&a, &b,
+		cmp.AllowUnexported(
+			consensus_channel.Guarantee{},
+			big.Int{},
+		),
+	)
+}
```

Comment:
If I understand correctly, these aren't used? Is it worth keeping them around?

Reply:
They are used in the new test file, yes.